.github/workflows/ci.yml

```diff
@@ -21,7 +21,8 @@ jobs:
         run: |
           wget https://apt.llvm.org/llvm.sh
           chmod +x llvm.sh
-          sudo ./llvm.sh 15
+          sudo ./llvm.sh 17
+          sudo apt install -y libpolly-17-dev
```

Comment:
NB: At some point LLVM appears to have dropped Polly from their default library base in `llvm.sh`. The `llvm-ir` crate needs it, so I've added it manually in each install step.